core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -489,6 +492,42 @@ public LineSenderBuilder autoFlushRows(int autoFlushRows) {
             return this;
         }
 
+
+        /**
+         * Set the interval at which the Sender automatically flushes its buffer.
+         * <br>
+         * It flushed the buffer even when the number of buffered rows is less than the value set by {@link #autoFlushRows(int)}.
+         * This prevents rows from being locally buffered for too long when the rate of incoming data is low.
+         * <p>
+         * <strong>Important:</strong>This option does not cause the Sender to flush the buffer at regular intervals.
+         * Auto-flushing is only triggered when calling {@link #atNow()}, {@link #at(Instant)} or {@link #at(long, ChronoUnit)}.
+         * The Sender will not flush the buffer if no new rows are added even if the auto-flush interval has elapsed.
+         * <p>
+         * This is only used when communicating over HTTP transport, and it's illegal to call this method when
+         * communicating over TCP transport.
+         * <br>
+         * You cannot set this value when auto-flush is disabled. See {@link #disableAutoFlush()}.
+         * <br>
+         * Default value is 1,000 milliseconds.
```

Comment:
1,000 doesn't read clearly across cultures where some interpret it as a decimal point.
I suggest 1'000 or simply 1000.